gpytorch/lazy/lazy_tensor.py

```diff
@@ -1692,36 +1700,21 @@ def root_inv_decomposition(self, initial_vectors=None, test_vectors=None, method
             else:
                 method = "lanczos"
 
+        # no need to do extra work if we already have a cached cholesky decomposition
```

Comment:
I think these comments and any commented code here is obsolete. Shall I go ahead and delete them?